tracer/test/test-applications/integrations/Samples.OracleMDA/Samples.OracleMDA.csproj

```diff
@@ -11,16 +10,22 @@
 
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Oracle.ManagedDataAccess" Version="$(ApiVersion)" />
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' ">
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.120"/>
+  </ItemGroup>
```

Comment:
Should we do multi-version testing here? 🤔

Reply:
yes, I wanted to eventually keep that for a later commit to get this test out first